test/libsolidity/smtCheckerTests/operators/shifts/compound_shift_right.sol

```diff
@@ -0,0 +1,30 @@
+pragma experimental SMTChecker;
+
+contract C {
+    function f(uint256 a, uint256 b) internal pure returns (uint256) {
+        a >>= b;
+        return a;
+    }
+    function t() public pure {
+        assert(f(0x4266, 0) == 0x4266);
+        // Fails because the above is true.
+        assert(f(0x4266, 0) == 0x426);
+
+        assert(f(0x4266, 0x8) == 0x42);
+        // Fails because the above is true.
+        assert(f(0x4266, 0x8) == 0x420);
+
+        assert(f(0x4266, 0x11) == 0);
+        // Fails because the above is true.
+        assert(f(0x4266, 0x11) == 1);
```

Comment:
@mijovic the cli tests are taking too long. I suspect it's because of the failing asserts in this example and the respective `left`. Failing asserts lead to counterexample generation which can take the solver a really long time, especially for more complicated cases. This doesn't necessarily look like such a case, but it actually is because it converts back and forth between SMT Integers and BitVectors, so that complicates the counterexamples as well. I'd suggest maybe removing this assertion and the similar one in the `left` case so we can see whether the cli time decreases.

Reply:
I did remove those checks, let's hope that it will pass now